### DISCUSSION TITLE
How does pybamm calculate stoichiometry range?

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3793] [Q&A]
x_0, x_100, y_100, y_0 = pb.lithium_ion.get_min_max_stoichiometries(param)
print("x_0: ", x_0, "x_100: ", x_100, "y_100: ", y_100, "y_0: ", y_0)

How does pybamm calculate this utilization windows and which parameters are utilised?

**Comment by valentinsulzer** [comment]
It is documented here: https://docs.pybamm.org/en/latest/source/examples/notebooks/models/electrode-state-of-health.html
